orgs/release.json

```diff
@@ -1,11 +1,11 @@
 {
   "orgName": "CumulusCI - Release Test Org",
   "edition": "Enterprise",
-  "orgPreferences" : {
-    "enabled": [
-      "S1DesktopEnabled",
-      "Translation",
-      "ChatterEnabled" 
-    ]
+  "settings": {
+    "orgPreferenceSettings": {
+      "s1DesktopEnabled": true,
+      "chatterEnabled": true,
+      "translation": true
```

Comment:
While we're working here, let's disable translation for the beta and release scratch orgs. It refers to translation workbench which is only needed for development. We had one bug where code was written that depended on translation workbench being installed (by referring to a permission that only exists in that case) and we didn't catch it until installing in an org with translation workbench turned off.